docrepr/webview.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*
+
+"""
+View object documentation HTML in a separate GUI window.
+
+Requires PyQt4 or PySide. Original code by
+anatoly techtonik <techtonik@gmail.com> is placed in
+public domain.
```

Comment:
But what's the code that *exactly* can be copy/pasted? I'd say the only important code is the one in `show_window`. The rest can't be really copy/pasted.

So then, please add a docstring and your name to that function only.